### DISCUSSION TITLE
HCPCS J or HCPCS Q codes with NDCs

### DISCUSSION THREAD

**Discussion by taylorpatriciab** [CMSgov/price-transparency-guide#165] [Q&A]
Many commercial carriers require professional and institutional claims submitted with drug-related (HCPCS J or HCPCS Q) codes include the National Drug Code (NDC) number, quantity and the unit of measure.  The NDC data allows for a more accurate payment and better management of drug costs based on what was administered and billed, minimizing claim appeals and adjustments.  They act similar to modifiers on CPT and HCPCS codes, adjusting the provider reimbursement rate based on the drug administered.  
Given these are drugs, and if reimbursed on a fee-for-services basis, should the contracted amounts be included in the prescription-drugs file or in the in-network-rates file?  If they should be reported on the prescription-drugs file, where should the HCPCS be reported?  If they should be reported on the in-network file, where should the NDC be reported?

**Comment by nmayle** [comment]
Great question. The use of both NDC and HCPCS codes for the same item occur frequently in files for Hospital Transparency files.
This schema for some reason does not seem to support having both codes. As to which file these items belong in, the rule for the in-network file seems to include all items: "the first file will show negotiated rates for all covered items and services between the plan or issuer and in-network providers."  Aren't many drugs covered when administered by hospitals, urgent care, ambulatory surgical centers, cancer centers, etc.?
